SeaPublicWebsite/Views/StaticPages/PrivacyPolicy.cshtml

```diff
@@ -0,0 +1,184 @@
+            We may also collect and process anonymised information as outlined in our
+            <a class="govuk-link" href="@Url.Action(nameof(CookieController.CookieSettings_Get), "Cookie")">
+                cookie policy
+            </a>.
+        </p>
+
+        <h2 class="govuk-heading-s">
+            Purpose
+        </h2>
+        <p class="govuk-body">
+            We are processing your personal data in order to receive feedback about this service and to contact 
+            you for clarifications or to report service updates.
+        </p>
+        
+        <h2 class="govuk-heading-s">
+            Legal basis of processing
+        </h2>
+        <p class="govuk-body">
+            The legal basis for processing your personal data is processing necessary for the performance of a task
+            carried out in the public interest or in the exercise of official authority vested in the data controller,
+            such as the exercise of a function of the Crown, a Minister of the Crown, or a government department; the
+            exercise of a function conferred on a person by an enactment; the exercise of a function of either House of
+            Parliament; or the administration of justice. In this case, in order to receive feedback about this service
+            and to contact you for clarifications or to report service updates.
+        </p>
+        
+        <h2 class="govuk-heading-s">
+            Recipients
+        </h2>
+        <p class="govuk-body">
+            Your personal data will be shared by us with other Government Departments or public authorities where
+            necessary. We may share your data if we are required to do so by law, forexample by court order or to
+            prevent fraud or other crime.
+        </p>
+        <p class="govuk-body">
+            As your personal data will be stored on our IT infrastructure it will also be shared with our data
+            processors Microsoft and Amazon Web Services.
+        </p>
+        <p class="govuk-body">
+            Your personal data may also be shared with the Softwire Technology Ltd who developed and help to maintain
+            this service.
+        </p>
+        
+        <h2 class="govuk-heading-s">
+            Retention
+        </h2>
+        <p class="govuk-body">
+            Your personal details will be kept for a period of up to six months following the date of your final email
+            sharing the details with us.
+        </p>
+        
+        <h2 class="govuk-heading-s">
+            Automated decision making
+        </h2>
+        <p class="govuk-body">
+            Your personal data will not be subject to automated decision making.
+        </p>
+        
+        <h2 class="govuk-heading-s">
+            Your Rights
+        </h2>
+        <p class="govuk-body">
+            You have the right to request information about how your personal data are processed, and to request a copy
```

Comment:
are processed -> is processed?

Reply:
Technically `are` is correct, but it does sound odd.